caches/infinispan/src/test/java/org/commonjava/maven/galley/cache/infinispan/FastLocalCacheProviderTest.java

```diff
@@ -161,7 +161,7 @@ public void testConstructorWitNFSSysPath()
     {
         System.setProperty( FastLocalCacheProvider.NFS_BASE_DIR_KEY, temp.newFolder().getCanonicalPath() );
         new FastLocalCacheProvider( new PartyLineCacheProvider( temp.newFolder(), pathgen, events, decorator ), cache,
-                                    pathgen, events, decorator, executor );
+                                    pathgen, events, decorator, executor, null );
```

Comment:
This null is for nfsBaseDir to test missing nfsBaseDir scenario. It is not the cache param. The cache param is using the Producer to get cache here.